rdkcentral/Lightning-UI-Components
# fix(Storybook): fixes Component Styles addon

## Description

this PR refactors the Components Style add-on
- breaks out ComponentsPanel file into React components for better readability and structure
- moves Tone, Color, Number components into components folder
- breaks out Component Style Table into it's own component
- adds `componentUtils.js` for functions shared between addon components
- adds`useState` (React) to Color and Number row to update and maintain proper values when clicking between panels 
- adds js documentation for better developer use: @params, @returns, etc.
- adds a Reset Styles button to panel to reset values back to original. NOTE: right now this manually setting to basic theme, need to map to theme that is selected in ThemePicker

## References
[LUI-1239](https://ccp.sys.comcast.net/browse/LUI-1239)


## Testing
- yarn clean && yarn install && yarn start
- go to story => componentStylePanel
- change a few styles
- click to Controls panel and then click back, styles should still be what you changed them to

## Automation


## Checklist
